abjad/tools/schemetools/Scheme.py

```diff
@@ -233,15 +238,15 @@ def format_embedded_scheme_value(value, force_quotes=False):
         '''
         from abjad.tools import datastructuretools
         from abjad.tools import schemetools
-        result = Scheme.format_scheme_value(value, force_quotes=force_quotes)
-        if isinstance(value, bool):
-            result = '#{}'.format(result)
-        elif isinstance(value, datastructuretools.OrdinalConstant):
-            result = '#{}'.format(repr(value).lower())
-        elif isinstance(value, str) and not force_quotes:
-            result = '#{}'.format(result)
-        elif isinstance(value, schemetools.Scheme):
-            result = '#{}'.format(result)
+        if isinstance(value, datastructuretools.OrdinalConstant):
+            result = '#' + repr(value).lower()
+        else:
+            result = schemetools.Scheme.format_scheme_value(
+                value, force_quotes=force_quotes)
+            if ((isinstance(value, bool)) or
+                    (isinstance(value, str) and not force_quotes) or
+                    (isinstance(value, schemetools.Scheme))):
+                result = '#' + result
```

Comment:
Anyways, I'll try to not get too hung up on this. There's going to be a lot of syntax twiddling to do, stale imports etc. everywhere. This codebase is ~10 years old.